File: _LibSDK/LibSDK/src/main/java/com/yline/base/BaseDialogFragment.java
package com.yline.base;

import android.os.Bundle;
import android.support.v4.app.DialogFragment;

import com.yline.application.BaseApplication;

public class BaseDialogFragment extends DialogFragment {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        BaseApplication.addFragmentForRecord(this);
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
        BaseApplication.removeFragmentForRecord(this);
    }
}

File: _LibSDK/LibSDK/src/main/java/com/yline/base/BaseDialogFragment.kt
package com.yline.base

import android.os.Bundle
import androidx.fragment.app.DialogFragment

import com.yline.application.BaseApplication

class BaseDialogFragment : DialogFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        BaseApplication.addFragmentForRecord(this)
    }

    override fun onDestroy() {
        super.onDestroy()
        BaseApplication.removeFragmentForRecord(this)
    }
}
